### ISSUE TITLE
Missed timestamp when running query for template variable population

### ISSUE THREAD

**Issue by roman-vynar** [grafana/grafana-plugins#42]
There is a way to create a dashboard variable of the type "query" under Templating.
However, when you put a query to populate possible values of your variable, Grafana sends this API request
```
http://grafana:3000/api/datasources/proxy/1/api/v1/query?query=up
```
and gets
```
{"status":"error","errorType":"bad_data","error":"cannot parse \"\" to a valid timestamp"}
```

The request is invalid as it misses "&time=<timestamp>".
So it should look like this:
```
http://grafana:3000/api/datasources/proxy/1/api/v1/query?query=up&time=1442488601
```

**Comment by roman-vynar** [comment]
datasource.js:
```
-         url = '/api/v1/query?query=' + encodeURIComponent(query);
+         var now = new Date();
+         url = '/api/v1/query?query=' + encodeURIComponent(query) + '&time=' + convertToPrometheusTime(now);
```

**Comment by torkelo** [comment]
time is not used when getting variable values. what datasource are you using?

**Comment by roman-vynar** [comment]
Sorry I didn't mention, it's Prometheus.

**Comment by roman-vynar** [comment]
Looks like it will be fixed by PR #41
